Sources/Sentry/SentryNetworkTracker.m

```diff
@@ -284,7 +284,9 @@ - (nullable NSDictionary *)addTraceHeader:(nullable NSDictionary *)headers
 
     id<SentrySpan> span = SentrySDK.currentHub.scope.span;
     if (span == nil) {
-        return headers;
+        NSMutableDictionary *existingHeaders = headers.mutableCopy;
+        [existingHeaders removeObjectsForKeys:@[ SENTRY_TRACE_HEADER, SENTRY_BAGGAGE_HEADER ]];
+        return [existingHeaders copy];
```

Comment:
Please just add a comment here why we removing the header. 
Because NSURLSession itself caches its, and were good to go.